extensions/azurecore/src/azureResource/providers/database/databaseTreeDataProvider.ts

```diff
@@ -44,7 +44,7 @@ export class AzureResourceDatabaseTreeDataProvider extends ResourceTreeDataProvi
 				databaseName: database.name,
 				userName: database.loginName,
 				password: '',
-				authenticationType: 'AzureMFA',
+				authenticationType: 'SqlLogin',
```

Comment:
Why are you changing these? Given that this is going through an Azure account isn't AzureMFA the more reasonable default to use?

Reply:
There are a few bugs with my approach here, I'm going to change this back to SqlLogin for May once my fixes are in.

Essentially this doesn't reuse your current connection and will keep prompting you for a password